src/lib/cookies.ts

```diff
@@ -38,13 +38,10 @@ const getCookieValues = (name: string): string[] => {
     }, []);
 };
 
-export const getCookie = (name: string): string | null => {
+export const getCookie = (name: string): string | undefined => {
     const cookieVal = getCookieValues(name);
 
-    if (cookieVal.length > 0) {
-        return cookieVal[0];
-    }
-    return null;
+    return cookieVal[0];
```

Comment:
This function was only being used in one location, which relied just checked that the return value was truthy. Seems easiest to just embrace `undefined` in the functions return type